debbie.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'hirb', '~> 0.7'
   gem.add_runtime_dependency 'coolline', '>= 0.4.2'
   gem.add_runtime_dependency 'awesome_print', '~> 1.2'
-  gem.add_runtime_dependency 'railties', '>= 3.0', '< 5.0'
+  gem.add_runtime_dependency 'railties', '>= 3.0', '< 5.2'
```

Comment:
Hi @Biggless thanks for this, if there are no issues on `railties 5.0.0` I thinks that's better to eliminate the upper restriction completely. So leave it like this:
```ruby
  gem.add_runtime_dependency 'railties', '>= 3.0'
```